tests/ia/README.md

```diff
@@ -0,0 +1,22 @@
+# **How to test Integrated Alerting**
+### Requirements :
+* PMM Server 2.19.0 or higher
+#### Environment variables:
+* MAILOSAUR_API_KEY
+* MAILOSAUR_SERVER_ID
+* MAILOSAUR_SMTP_PASSWORD
+* PAGER_DUTY_SERVICE_KEY
+* PAGER_DUTY_API_KEY
+### Command to execute Integrated Alerting Alerts tests command
+
+`npx codeceptjs run -c pr.codecept.js tests/ia/alerts_test.js --steps`
+
+### Command to execute all Integrated Alerting tests:
+
+`npx codeceptjs run -c pr.codecept.js --grep '@ia'`
+
+### Execute Integrated Alerting tests in Jenkins
```

Comment:
lovely

Reply:
may be setting up webhook on local, not using the test setup, just instructions on how many compose I need to start

Reply:
webhook start on local is done automatically from the BeforeSuite hook. So no need to do anything from user perspective

Reply:
I meant to say, if I just needed the setup, without having to run the test, could I use it somehow, my objective is to get a ready environment with webhook via staging, so I could pass it like a flag, or add the same steps with framework

Reply:
done